### DISCUSSION TITLE
`author`, `contributors` fields in `package.json`

### DISCUSSION THREAD

**Discussion by ctmbl** [iScsc/iscsc.fr#52] [General]
*started [here](https://github.com/iScsc/iscsc.fr/pull/50#issuecomment-1323896622)*

@atxr 
> Aha I'm totally fine about being the author of this repo, even if it's the whole club

Actually I was suggesting the opposite: as more contributors will add their work to the repo we could add their name to the `contributor` field as suggested by the [documentation](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors)

**Comment by amtoine** [comment]
and thus create a list of contributors, right? @ctmbl

**Comment by dolphinau** [comment]
Of course, adding each of us in the contributors is way better!
And this also solves the mail discussion: we can put supaerocsclub@gmail.com

**Comment by ctmbl** [comment]
> and thus create a list of contributors, right? @ctmbl

That's right!

> Of course, adding each of us in the contributors is way better!

Then we'll think about that in next PRs

> And this also solves the mail discussion: we can put [supaerocsclub@gmail.com](mailto:supaerocsclub@gmail.com)

yeah!
